app/(tabs)/account/Login.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import { Alert, Text, TextInput, View } from "react-native";
+
+import { Button } from "tamagui";
+
+import { commonStyles as styles } from "./styles";
+
+interface LoginProps {
+  username: string;
+  password: string;
+  setIsCreatingAccount: (isCreatingAccount: boolean) => void;
+  setIsLoggedIn: (isLoggedIn: boolean) => void;
+}
+
+export default function Login({
+  username,
+  password,
+  setIsCreatingAccount,
+  setIsLoggedIn,
+}: LoginProps) {
+  const [inputUsername, setInputUsername] = useState<string>(username);
+  const [inputPassword, setInputPassword] = useState<string>(password);
+
+  const handleLogin = () => {
+    if (inputUsername === username && inputPassword === password) {
+      setIsLoggedIn(true);
+    } else {
+      Alert.alert("Invalid credentials", "Please try again.");
```

Comment:
Usually this would be displayed as text above/below the form instead of a popup, but this is fine for the prototype